hackathon_site/hardware/test_api.py

```diff
@@ -508,7 +535,7 @@ def test_user_has_view_permissions(self):
         self.assertEqual(expected_response, data["results"])
 
```

Comment:
One more test you could add: Make sure that when a user has a profile and view permissions, they can still see everything.